src/Blameable/Mapping/Driver/Annotation.php

```diff
@@ -20,6 +20,8 @@
  * extension.
  *
  * @author David Buchmann <mail@davidbu.ch>
+ *
+ * @final since gedmo/doctrine-extensions 3.x
  */
 class Annotation extends AbstractAnnotationDriver
```

Comment:
When adding support for attributes, this `Annotation` class would be extended by an `Attribute` class to not have duplicated code, maybe we should mark as internal some of them? like the drivers?